### PR TITLE
Add shortcuts docs

### PR DESCRIPTION
**Pull request by seratch**
###  Summary

In addition to #401 , this pull request updates the document to have the shortcuts feature.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/java-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: docs/guides/shortcuts.md [hunk 1]
```diff
@@ -1,37 +1,32 @@
...
 ---
 
-# Actions
+# Shortcuts
 
-[Actions](https://api.slack.com/interactivity/actions) are simple shortcuts that people can use to quickly complete a task with your app — like reporting a bug, requesting time off, or starting a meeting.
+Shortcuts are an evolution of slash commands, surfaced in both the composer menu and the quick switcher. Shortcuts allow users to trigger your app's workflows from intuitive surface areas within Slack.
 
-As of March 2020, this SDK supports the following features (yes, there are [more to come soon](https://medium.com/slack-developer-blog/introducing-the-slack-app-toolkit-3d509a15f41b)!).
+For all types of shortcut requests, your app has to respond to the request within 3 seconds by `ack()` method. Otherwise, the user will see the timeout error on Slack.
```
Comment (shaydewael) [inline docs/guides/shortcuts.md]: Suggestion: Your app has 3 seconds to call `ack()`, which acknowledges a shortcut request is received from Slack.

File: docs/guides/shortcuts.md [hunk 2]
```diff
@@ -1,37 +1,32 @@
...
-
-For all types of action requests, your app has to respond to the request within 3 seconds by `ack()` method. Otherwise, the user will see the timeout error on Slack.
-
-## Message Actions
+## Global / Message Shortcuts
 
 ### Slack App Configuration
 
-To enable Message Actions, visit the [Slack App configuration page](http://api.slack.com/apps), choose the app you're working on, and go to **Features** > **Interactive Components** on the left pain. There are four things to do on the page.
+To enable global/message shortcuts, visit the [Slack App configuration page](http://api.slack.com/apps), choose the app you're working on, and go to **Features** > **Interactive Components** on the left pain. There are four things to do on the page.
```
Comment (shaydewael) [inline docs/guides/shortcuts.md]: global/message shortcuts -> shortcuts

File: docs/guides/shortcuts.md [hunk 3]
```diff
@@ -48,16 +43,27 @@ Bolt does many of the commonly required tasks for you. The steps you need to han
 * Build a reply message or do whatever you want to do
 * Call `ack()` as an acknowledgment
 
-Message actions request payloads have `response_url`, so that your app can reply to the action (even asynchronously after the acknowledgment). The URL is usable up to 5 times within 30 minutes of the action invocation. If you post a message using `response_url`, call `ctx.ack()` without arguments and use `ctx.respond()` to post a message.
+Message shortcut request payloads have `response_url`, so that your app can reply to the shortcut (even asynchronously after the acknowledgment). The URL is usable up to 5 times within 30 minutes of the shortcut invocation. If you post a message using `response_url`, call `ctx.ack()` without arguments and use `ctx.respond()` to post a message. Global shortcut request payloads don't have `response_url`.
```
Comment (shaydewael) [inline docs/guides/shortcuts.md]: Might be help to mention `response_url_enabled` field for `conversation_select` elements. https://api.slack.com/reference/block-kit/block-elements#conversation_select. Using that in a modal is the recommended way to post a message.

Although if you don't feel like it fits into this section, that's completely fine.

**Comment by seratch** [comment]
@shaydewael Thanks as always! I will update the document reflecting your feedback after finishing some works on the implementation side.

**Pull request merged**